main.cpp

```diff
@@ -5,17 +5,24 @@
 #include <opencv2/opencv.hpp>
 #include <opencv2/imgcodecs.hpp>
 #include "src/LaneDetection/detect.hpp"
+#include "src/LaneKeeping/lanekeeping.hpp"
+
+#define FRAME_WIDTH 510
+#define FRAME_HEIGHT 280
 
 
 int main()
 {
 
     LaneDetectionResults ld_results;
-    if (true)
+    bool has_video = true;
+    if (has_video)
     {
         std::string camera("455");
-        std::string video_path = cv::String("path/video_repository/highway.mp4");
+        std::string full_path_to_videos = "/mnt/c/Users/giann/Desktop/Work Experience/LaneAssist/"; // put your full path to repo
+        std::string video_path = cv::String(full_path_to_videos + "video_repository/highway.mp4");
```

Comment:
i like this one.

Reply:
Now you will like it more. 
I'm using the same param in the image section.